README.md

```diff
@@ -1 +1,93 @@
+
+## Project technology stack
+
+- Faker - for generate words
+- React cookies - for using cookies in React (save user authorization)
+- JWT - for generate user credential 
+- i18next - make our application with multi-language 
+- Pagination - for make pagination on LeadersBoard page
+- MaterialUi - for styling some components 
+- Simple keyboard - for showing helpfully keyboard
+- NodeJs - for [back-end](https://github.com/HKudria/rsCloneBackend)
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `dist` folder.\
+It correctly bundles React with Vite in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## Learn More
+
+You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
+You can learn more in the [Vite documentation](https://github.com/vitejs/vite/blob/main/README.md).
+
+To learn React, check out the [React documentation](https://reactjs.org/).
+
+## Running ESLint
+
+Inside the project directory, open a terminal.
+
+To run ESLint and see what errors it is pointing, just run:
+
+`npm run lint`
+
+To automatically fix some errors, you can use ‘--fix’:
+
+`npm run lint:fix`
+
+If you want to ignore warnings, you can use ‘ --quiet’
+
+`npm run format`
+
+To learn more [Install eslint in React Project](https://andrebnassis.medium.com/setting-eslint-on-a-react-typescript-project-2021-1190a43ffba)
+
+## MaterialUi
+
+To learn more [MaterialUi Documentation](https://mui.com/)
+
+## Faker
+
+To learn more [Faker.js Documentation](https://github.com/faker-js/faker)
+
+## Simple keyboard
+
+To learn more [Simple keyboard Documentation](https://github.com/simple-keyboard)
+
+## Pagination
+
+To learn more [Pagination Documentation](https://www.npmjs.com/package/react-paginate)
+
+## i18next
+
+To learn more [i18next Documentation](https://github.com/i18next/i18next)
+
+## react-cookie
+
+To learn more [react-cookie Documentation](https://github.com/reactivestack/cookies/tree/master/packages/react-cookie/)
```

Comment:
## nivo/pie
To learn more [nivo/pie Documentation](https://nivo.rocks/pie/)

## react-typing-game-hook-v2 
To learn more [ react-typing-game-hook-v2  Documentation](https://github.com/jokarz/react-typing-game-hook)